src/app/clusters/door-lock-server/door-lock-server-user.c

```diff
@@ -609,7 +612,9 @@ static void startLockout()
     uint8_t lockoutTimeS = getUserCodeTemporaryDisableTime();
     if (lockoutTimeS != 0)
     {
-        emberEventControlSetDelayMS(emberAfPluginDoorLockServerLockoutEventControl, lockoutTimeS * MILLISECOND_TICKS_PER_SECOND);
+        /*emberEventControlSetDelayMS(emberAfPluginDoorLockServerLockoutEventControl, lockoutTimeS * MILLISECOND_TICKS_PER_SECOND);
+         * Replace silab emberEventControlSet (DelayMS/Inactive...) methods with equivalent #2503
+         */
```

Comment:
Can we uncomment that ?

Reply:
yes, should be able to uncomment these now